src/main.ts

```diff
@@ -50,6 +52,39 @@ export default function (): void {
 		return allowFile;
 	});
 
+	LocalMain.HooksMain.addAction('importSiteChangeDomain', async (site: Site) => {
+		const { wpCli, localLogger } = serviceContainer.cradle;
+
+		if (site.getSiteServiceByRole(Local.SiteServiceRole.FRONTEND)) {
+			try {
+				const faustWPsettings = await wpCli.run(site, [
+					'option',
+					'get',
+					'faustwp_settings',
+					'--format=json',
+				]);
+
+				const parsedFaustWPsettings = JSON.parse(faustWPsettings);
+
+				if (parsedFaustWPsettings.frontend_uri !== site.frontendUrl) {
```

Comment:
I like that you've scoped our actions to only the `FRONTEND` service. 

One thing that comes to mind -- If we've made it this far in our logic, then we know that the `site.frontendUrl` is different than what it should be in the DB.

Because of this, do we need to run an update on that option using WPCLI? 

What if instead, we do a more generic search/replace that tries to update the url _everywhere_ in the DB?

Maybe something that uses `SearchReplacerWPDatabaseDomain()` like this:

As an added benefit, this might address [LOC-3213](https://wpengine.atlassian.net/browse/LOC-3212)